covid_graphs/covid_graphs/prediction_generator.py

```diff
@@ -7,8 +7,8 @@
 from google.protobuf import text_format  # type: ignore
 
 from .country_report import CountryReport, create_report
-from .formula import FittedFormula
-from .pb.atg_prediction_pb2 import AtgParameters, CountryAtgParameters
+from .formula import fit_country_data
```

Comment:
Nit: I'm a fan of `from . import formula` and then `formula.fit_country_data` since it makes it clear the function comes from a different module.

Reply:
Done